Controllers/FamilyController.cs

```diff
@@ -235,15 +257,21 @@ private Task CreateMembersAndConsiderations(FamilyViewModel Family)
                     ConsiderationsCreateDto cuisine =
                         new()
                         {
-                            MemberId = CreatedMember.MemberId,
+                            MemberId = CreatedMember.Data!.MemberId,
```

Comment:
all of the `!` used here are after we know that we've successfully made the call and got back a successful serviceResult